### DISCUSSION TITLE
el PBTaskAddEditor arroja un error de <!DOCTYPE html> con accessRole

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#337] [4. Bug Reports] [answered]
Hola

En referencia a las discusiones del PBTaskAddEditor
https://github.com/Cotalker/documentation/discussions/249
https://github.com/Cotalker/documentation/discussions/295
https://github.com/Cotalker/documentation/discussions/307

Resulta que en una rutina, creo una task y agrego a usuarios con rol determinado y posteriormente mando un mensaje al canal con el link de la tasks. Resulta que cuando terminé de configurar la rutina y hago un QA, la rutina se detiene en la parte del PBTaskAddEditor y al revisar en los logs me arroja esto:

```
11:
key: "add_broncpulmonar"
name: "PBTaskAddEditor"
status: "ERROR"
result:
errors:
0: "STATUS CODE: 500 BODY: <!DOCTYPE html> <html lang="en"> <head> <meta charset="utf-8"> <title>Error</title> </head> <body> <pre>Internal Server Error</pre> </body> </html> "
nextBot: ""
iteration: 12
```
No entiendo mucho que significa (quizás hay algo comiteado que faltó lanzarse)

Dejo la rutina del bot indicado quizás tenga algo que falle.

```
{
                "_id" : ObjectId("60f0449147b7610009606e6d"), 
                "key" : "add_broncpulmonar", 
                "name" : "PBTaskAddEditor", 
                "version" : "2.0.0", 
                "data" : {
                    "type" : "accessRole", 
                    "accessRole" : "broncopulmonar", 
                    "task" : "$OUTPUT#crear_paciente_broncopulmonar#task|_id", 
                    "taskGroup" : "5e0a1a1345e9602068206fdb"
                }, 
                "next" : {
                    "SUCCESS" : "send_paciente_bronco", 
                    "ERROR" : ""
                }, 
                "customNetworkRequest" : [

                ]
            }
```
Dejo tambien el id del bot en prod: 60dcdfd2aad21d000818d028 por si se requiere alguna revisión. Me podrías ayudar a revisar @guillevalin ?

Muchas gracias :>

**Comment by acedward** [comment, marked as answer]
Nos puedes mandar por interno los 3 logs de "automation logs"

**Reply by NicolasPLG** [reply]
los mando

**Reply by NicolasPLG** [reply]
Mientras revisaba por base de datos, me di cuenta que en el campo "**taskGroup**" había puesto el **id del user system** y no el **id de la taskgroup** que correspondía, generando ese error de texto

Lo más fácil era colocar $OUTPUT#**create_task**#task|taskGroup pero el id es más fácil de recordar
